### PR TITLE
Add Login card.

### LINKED ISSUE #11: Create Login Component
This component should have built-in form validation, and should sent a POST request to a register endpoint defined by the backend.

### PR DESCRIPTION
**Pull request by suneettipirneni**
closes #11

### CODE REVIEW

File: src/components/landing/LoginCard.jsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    event.preventDefault();
+
+    // TODO API
+  }
+
+  return (
+    <LandingCard title="Welcome Back">
+      <form onSubmit={login}>
+        <TextInput placeHolder="username" key="Username" />
+        <TextInput placeHolder="password" key="Password" type="password" />
```
Comment (ctcuff) [inline src/components/landing/LoginCard.jsx]: Could you capitalize `username` and `password` for the placeholder? That's just to match the capitalization in the other PRs.

File: src/components/TextInput.jsx [hunk 2]
```diff
@@ -33,6 +35,7 @@ TextInput.propTypes = {
   value: PropTypes.string,
   onChange: PropTypes.func,
   type: PropTypes.oneOf(['email', 'password', 'text']),
+  key: PropTypes.string,
```
Comment (ctcuff) [inline src/components/TextInput.jsx]: A prop named `key` won't be forwarded to the component (`key` will always be undefined here) so you may want to rename this. Maybe something like `name` and then `key={name}`? (Naming is up to you honestly)

**Review by ctcuff — CHANGES_REQUESTED** [review]
Few small changes but otherwise looks great! For styling, I'd also like to suggest `100%` for the width of the login button and a little bottom margin for the inputs since they seem a little close (but maybe we should wait for #20 since it changes the layout of the input a bit?)

File: src/components/TextInput.jsx (hunk 2 above)
Comment (suneettipirneni) [reply src/components/TextInput.jsx]: I'm not sure I follow, isn't key passed to the input element?

File: src/components/landing/LoginCard.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+function LoginCard() {
+  async function login(event) {
+    event.preventDefault();
+
+    // TODO API
+  }
+
+  return (
+    <LandingCard title="Welcome Back">
+      <form onSubmit={login}>
```
Comment (ElbertTse) [inline src/components/landing/LoginCard.jsx]: Consider adding the card-form class to the form element so the button and text input take up the same width. That was how I did it for the group login card but if the register card does not do that then we can go either way in terms of making the style consistent among cards.

File: src/components/TextInput.jsx (hunk 2 above)
Comment (ctcuff) [reply src/components/TextInput.jsx]: Sorry, I don't think I explained very well! `key` is a special prop: https://reactjs.org/warnings/special-props.html, but basically, doing `console.log({ key })` gives `{ key: undefined }` in the console.

**Review by ElbertTse — CHANGES_REQUESTED** [review]
See the comment I left but if the register card does not include that class then we can discuss whether we want to include the card-form class or remove it.

File: src/components/TextInput.jsx (hunk 2 above)
Comment (suneettipirneni) [reply src/components/TextInput.jsx]: Ah ok! I'll fix that

**Pull request merged**